skopt/optimizer/optimizer.py

```diff
@@ -171,7 +169,9 @@ def __init__(self, dimensions, base_estimator="gp",
                  model_queue_size=None,
                  acq_func_kwargs=None,
                  acq_optimizer_kwargs=None):
-        self.specs = {"args": copy.copy(inspect.currentframe().f_locals),
+        args = locals()
```

Comment:
I guess the function call constructs a dict on the fly, but I'd do a `locals().copy()` to comply with the [note here](https://docs.python.org/3/library/functions.html#locals).